multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/cf/CloudControllerClientProvider.java

```diff
@@ -21,8 +18,10 @@
 @Named
 public class CloudControllerClientProvider {
 
+    private static final String NO_CORRELATION_ID = "NO-ID"; // TODO decide better dummy value
```

Comment:
You will use this constant only to build key which will be used only in cache map. It won't be used to add correlation id header with this constant, right?

Reply:
Yes, that is right. You may propose better value if anything comes to your mind.

Reply:
I am fine with this value